slee/slee.md

```diff
@@ -0,0 +1,13 @@
+# FirstName : Seungwoo, LastName : Lee (slee) 
+## Profile Pic: 
+## Call me by this name: Lee
+## Cohort: July 2023
+## Email ID: lmk2215@gmail.com 
+## LinkedIn: 
```

Comment:
Please consider creating LinkedIn profile, if you explicitly do not want to create one, then reply to this comment by saying, " I don't want to create a LinkedIn profile"

Reply:
Ok, I made a new Linkedin profile